algorithms/src/snark/marlin/ahp/prover/round_functions/mod.rs

```diff
@@ -30,21 +30,22 @@ use snarkvm_utilities::println;
 #[cfg(feature = "parallel")]
 use rayon::prelude::*;
 
+mod fifth;
 mod first;
 mod fourth;
 mod second;
 mod third;
```

Comment:
Nit: let's sort this by round (if rustfmt re-orders things, let's use a `#[rustfmt::skip]`

Reply:
to do this we'd either have to bury them in a separate mod (so that we can scope the directive), or tag each line with `#[rustfmt::skip]` - if there is no problem adding an extra mod layer then i'll go for that!